packages/data-api/src/fetchData.js

```diff
@@ -49,63 +73,24 @@ const getA1WhereClause = conditions => {
   return clause;
 };
 
-const getA1GroupByClause = firstAggregationType => {
-  switch (firstAggregationType) {
-    case 'FINAL_EACH_DAY':
-      return `GROUP BY ${COMMON_FIELDS}, day_period`;
-    case 'FINAL_EACH_WEEK':
-      return `GROUP BY ${COMMON_FIELDS}, week_period`;
-    case 'FINAL_EACH_MONTH':
-      return `GROUP BY ${COMMON_FIELDS}, month_period`;
-    case 'FINAL_EACH_YEAR':
-      return `GROUP BY ${COMMON_FIELDS}, year_period`;
-    case 'MOST_RECENT':
-      return `GROUP BY ${COMMON_FIELDS}`;
-    default:
-      return '';
-  }
-};
-
-const getA2FinalWhereClause = firstAggregationType => {
-  switch (firstAggregationType) {
-    case 'FINAL_EACH_DAY':
-      return 'AND day_period = a1.day_period';
-    case 'FINAL_EACH_WEEK':
-      return 'AND week_period = a1.week_period';
-    case 'FINAL_EACH_MONTH':
-      return 'AND month_period = a1.month_period';
-    case 'FINAL_EACH_YEAR':
-      return 'AND year_period = a1.year_period';
-    default:
-      return '';
-  }
+const getA1GroupByClause = firstAggregation => {
```

Comment:
Should we name this `aggregation` or `firstAggregation` in this method?

Reply:
I've been calling it `firstAggregation` everywhere to indicate that it's just the first one in the aggregation list. I'm not too fussed either way though?

Reply:
I'm happy with it too. Just a minor thought 😄